core/src/main/java/org/jdbi/v3/core/statement/Batch.java

```diff
@@ -60,7 +61,13 @@ public Batch add(String sql) {
         try {
             try {
                 stmt = getHandle().getStatementBuilder().create(getHandle().getConnection(), getContext());
+
                 addCleanable(stmt::close);
+
+                Integer queryTimeout = getConfig(Connections.class).getQueryTimeout();
```

Comment:
Consider, should `Batch` et al actually understand what's going on here, or should we add a `protected void customize(Statement stmt)` that does the actual work to the config class?